docs/docs/your-fable-project/author-a-fable-library.md

```diff
@@ -18,7 +20,89 @@ The last point may sound complicated but it's only a matter of adding a couple o
 </ItemGroup>
 ```
 
-That's all it takes to make your library compatible with Fable! In order to publish the package to Nuget check [the Microsoft documentation](https://docs.microsoft.com/en-us/nuget/quickstart/create-and-publish-a-package-using-the-dotnet-cli) or alternatively you can also [use Fake](https://fake.build/dotnet-nuget.html#Creating-NuGet-packages).
+In order to publish the package to Nuget check [the Microsoft documentation](https://docs.microsoft.com/en-us/nuget/quickstart/create-and-publish-a-package-using-the-dotnet-cli) or alternatively you can also [use Fake](https://fake.build/dotnet-nuget.html#Creating-NuGet-packages).
+
+## Make your package discoverable
+
+[Fable.Packages](https://fable.io/packages/) is a tool making it easy for users to search for Fable packages. 
+
+To make your packages listed on Fable.Packages, you need to add some tags to your `.fsproj`.
+
+<ul class="textual-steps">
+
+<li>
+
+**All Fable** packages must have the `fable` tag.
+
+</li>
+
+<li>
+
+Specify what kind of package you are publishing.
+
+A fable package can be one of the following:
+
+- `fable-library`: A library that can be used in Fable
```

Comment:
"fable-library" is already used for the files Fable embeds in every application. Can we change the tag? Or maybe we can skip this tag entirely as it just means it's not a "pure" binding.

Reply:
> Or maybe we can skip this tag entirely as it just means it's not a "pure" binding.

We need to way to search for "Library" type packages. That's why there is this tag.

In theory because there are only 2 types of packages, we can consider that by default a package if a library unless there is `fable-binding` flag which make it a binding.

However, currently there are 365packages decorated with just `fable` tag and so it means that all of them are consider "libraries" right now. So the filter `Package type: Library` is basically useless until most of the packages applies the `fable-binding` tag accordingly.

Reply:
@alfonsogarciacaro 

After more thinking, I don't think I like having a tag for binding and the absence of that tags means it is a library.

You know me, I like when things are explicit and for explanation to people I think it is easier to say:

1. Also put `fable`
2. Choose between `fable-library` or `fable-binding`
3. Choose which target are supported `fable-javascript`, `fable-python`, etc.

`fable-library` the tag and the folder are 2 different things seen in 2 different context, so I don't think this is a problem.

We are already speaking all the time of `library` / `binding` when creating packages.